airbyte-webapp/src/components/ContentCard/ContentCard.tsx

```diff
@@ -14,7 +14,7 @@ interface IProps {
 const Title = styled(H5)<{ light?: boolean }>`
   padding: ${({ light }) => (light ? "19px 20px 20px" : "25px 25px 22px")};
   color: ${({ theme }) => theme.darkPrimaryColor};
-  box-shadow: ${({ light, theme }) => (light ? "none" : `0 1px 2px ${theme.shadowColor}`)};
+  border-bottom: #e8e8ed 1px solid;
```

Comment:
Does it also apply the "light" mode?

Reply:
updated to have no border bottom when theme is light.